core/src/main/java/org/commonjava/indy/core/expire/ScheduleManager.java

```diff
@@ -750,22 +750,26 @@ public void expired( CacheEntryExpiredEvent<ScheduleKey, Map> e )
         if ( !e.isPre() )
         {
             final ScheduleKey expiredKey = e.getKey();
+/*
```

Comment:
So, does this just stop using the locking cache? Will this still work if we deploy in a clustered environment, and we need to avoid trying to take actions on multiple nodes when a particular scheduled event expires?

Reply:
I guess we need to only let one node to use the event. We need a follow-up issue to fix that.

Reply:
I filed NOS-1906 to track.